interface/themes/style_pink.css

```diff
@@ -1,7 +1,7 @@
 /**
  * Stylesheet for flat colors
  * The color palette is used to style the various elements in a consistent manner
```

Comment:
brought this script changes into codebase
(note I removed the end of line whitespaced)